BitmovinConvivaAnalytics/Classes/ContentMetadataBuilder.swift

```diff
@@ -85,6 +93,9 @@ class ContentMetadataBuilder: CustomStringConvertible {
             if let custom = self.custom {
                 contentInfo.merge(custom) { $1 }
             }
+            if let additionalStandardTags = self.additionalStandardTags {
```

Comment:
Note: (no action required)

Since Swift 5.7 you could write this like this:
```suggestion
            if let additionalStandardTags {
```

However, the Swift version specified in the podspec file is set to `5.0`. We could update it since it's quite old already (we are on 5.10 no in the Player SDK) _Not relevant for this PR_.

Reply:
Ah cool, didn't know about that Swift feature 👍